docs/ios/features/feature_layout_snapshots.md

```diff
@@ -0,0 +1,21 @@
+# Feature — Layout Snapshots
+
+Layout snapshots provide a lightweight way to capture the structure of your UI at key user interactions.
+They are automatically collected during click events (with throttling) and store the layout hierarchy as SVG rather than
+full screenshots.
+This approach gives valuable context about the UI state during user interactions while being significantly more
+efficient to capture and store than traditional screenshots.
+
+| Screenshot                                            | Layout snapshot                                  |
+|-------------------------------------------------------|--------------------------------------------------|
+| ![Screenshot](../images/layoutSnapshotScreenshot.png) | ![Layout Snapshot](../images/layoutSnapshot.svg) |
```

Comment:
The image size difference between screenshot and layout snapshot is too large. Try making them the same dimensions. Otherwise it looks like this:

<img width="1920" alt="image" src="https://github.com/user-attachments/assets/2e70b8ec-5f25-461b-a922-2aaeea9e8b88" />